README.markdown

```diff
@@ -95,6 +95,46 @@ end
 
 See `lib/fakefs/spec_helpers.rb` for more info.
 
+### FakeFs vs `pp` --- `TypeError: superclass mismatch for class File`
+
+`pp` and `fakefs` collide, `require 'pp'` then `require 'fakefs'`.
+
+Mocking existing files
+----------------------
+
+If you want to use existing files in your fake fs, clone them first.
+
+```ruby
+it "can use existing files" do
+  FakeFS.activate!
```

Comment:
should this use the block style instead ? ... otherwise that is a global change ?